lib/dice_page.dart

```diff
@@ -1,101 +1,128 @@
+                      fontSize: Constants.answerFontSize,
                     ),
                   ),
-                  style: ElevatedButton.styleFrom(
-                    shape: RoundedRectangleBorder(
-                      borderRadius: BorderRadius.circular(
-                        Constants.playButtonBorderRadius,
-                      ),
+                  Container(
+                    margin: EdgeInsets.all(
+                      Constants.dicesContainerMargin,
                     ),
-                    primary: Colors.blue,
-                    padding: EdgeInsets.fromLTRB(
-                      Constants.playButtonTopBottomPadding,
-                      Constants.playButtonLeftRightPadding,
-                      Constants.playButtonTopBottomPadding,
-                      Constants.playButtonLeftRightPadding,
+                    child: _buildImage(snapshot),
+                  ),
+                  Container(
+                    child: ElevatedButton(
+                      child: Text(
+                        'Good luck!',
+                        textAlign: TextAlign.center,
+                        style: TextStyle(
+                          color: Colors.black,
+                          fontWeight: FontWeight.bold,
+                          fontSize: Constants.playButtonFontSize,
+                        ),
+                      ),
+                      style: ElevatedButton.styleFrom(
+                        shape: RoundedRectangleBorder(
+                          borderRadius: BorderRadius.circular(
+                            Constants.playButtonBorderRadius,
+                          ),
+                        ),
+                        primary: Colors.blue,
+                        padding: EdgeInsets.symmetric(
+                          horizontal: Constants.playButtonLeftRightPadding,
+                          vertical: Constants.playButtonTopBottomPadding,
+                        ),
+                      ),
+                      onPressed: () {
+                        widget.bloc.diceChanger();
+                      },
                     ),
                   ),
-                  onPressed: () {},
-                ),
+                ],
               ),
-            ],
-          ),
-        ),
+            ),
+          );
+        },
       ),
     );
   }
+
+  Widget _buildImage(AsyncSnapshot snapshot) {
+    if (snapshot.hasData) {
+      _assetList = _buildList(snapshot);
+      return ListView.builder(
+        itemCount: _assetList.length,
+        shrinkWrap: true,
+        itemBuilder: (BuildContext context, int index) {
+          return _assetList[index];
+        },
+      );
+    } else {
+      return Container();
+    }
+  }
+
+  List<Widget> _buildList(AsyncSnapshot snapshot) {
+    var _list = snapshot.data.map<Widget>((assetImage) {
+      return Image(
+        image: assetImage,
+        width: Constants.dicesWidthAndHeight,
```

Comment:
Rename this constant to a more generic name that not reference where you are using it , in this case you use for (width and height) you can name it like imageSize.